### PR TITLE
Response ruby-diagnostic - Larson

### PR DESCRIPTION
**Pull request by rlarsonil71**
Comfort: 4
Clarity: 4

I used class notes and the various intro Ruby trainings to complete this diagnostic.

### CODE REVIEW

File: lib/diagnostic.rb [hunk 1]
```diff
@@ -8,21 +8,21 @@
...
-Diagnostic.repl = 'your response here'
+Diagnostic.repl = 'pry'
 
 # Naming conventions in Ruby
 #
 # In a Ruby string, write what you would write to properly name a variable in
 # Ruby that read in English as "star wars episode viii".
 
-Diagnostic.variable = 'your response here'
+Diagnostic.variable = 'star wars episode viii'
```
Comment (jordanallain) [inline lib/diagnostic.rb:25]: Remember in Ruby we use snake_case:

`star_wars_episode_viii`

File: lib/diagnostic.rb [hunk 2]
```diff
@@ -93,13 +93,13 @@
 # In a Ruby string, write how you would remove the last two elements from this
 # array
 
-Diagnostic.remove_from_array = 'your answer here'
+Diagnostic.remove_from_array = 'arr.slice!(arr.length-1)'
```
Comment (jordanallain) [inline lib/diagnostic.rb:96]: Hmm, I think this would work but why not just use `arr.pop 2`

**Comment by jordanallain** [comment]
14/15

Fantastic work Ron!

🇯🇴 🇯🇴 🇯🇴 🇯🇴 🇯🇴 🇯🇴 🇯🇴 🇯🇴 🇯🇴 🇯🇴 🇯🇴 🇯🇴 🇯🇴 🇯🇴

**Comment by rlarsonil71** [comment]
Thanks Jordan!

**Pull request closed without merging**